packages/vue-spec-renderer/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import type { App } from 'vue'
+import VueSpecRenderer from './components/VueSpecRenderer.vue'
+
+export default {
+  install: (app: App): void => {
+    app.component('VueSpecRenderer', VueSpecRenderer)
+  }
+}
+
+export { VueSpecRenderer }
```

Comment:
suggestion: Since this component comes with some heavy dependencies, I'd recommend removing the Vue plugin export and only allowing usage via importing the component itself so that it's properly tree-shaken.
```suggestion
export default VueSpecRenderer
```

This also means you'll need to update the import and init in `sandbox/index.ts` to something like this:

```ts
import { createApp } from 'vue'
import App from './App.vue'
import VueSpecRenderer from '../src'

const app = createApp(App)

app.component(VueSpecRenderer)
app.mount('#app')
```

When you write the documentation for the component, you'll actually want to instruct users to import in the component they are using the lib in, not in the entry file (otherwise again, no tree-shaking).